lte/gateway/python/magma/pipelined/openflow/flows.py

```diff
@@ -411,7 +407,8 @@ def get_add_resubmit_next_service_flow_msg(datapath, table, match,
     if copy_table:
         actions.append(parser.NXActionResubmitTable(table_id=copy_table))
 
-    actions = actions + reset_scratch_reg_actions
+    if (reset_default_register == 0):
```

Comment:
Then here you'll have if reset_default_register:

Reply:
Changes done